File: run/ShowData.py
import os
from pylab import *


# set path
DATA_PATH ='../'

# load grid
MESHX=mlab.load(DATA_PATH+'phyGrid.meshX');
MESHY=mlab.load(DATA_PATH+'phyGrid.meshY');

subplot(1,1,1);
DATA_ZERO = zeros(MESHX.shape,float)
contourf(MESHX,MESHY,DATA_ZERO,colors = '0.75');
hold(True); 

# show potential lines
if os.path.isfile(DATA_PATH+'scalar.dat'):
    DATA_POT=mlab.load(DATA_PATH+'scalar.dat');

    subplot(1,1,1);

    contour(MESHX,MESHY,DATA_POT,20);
    title(' potential - isolines ');
    #colorbar(cpot)
    axis('equal')
    hold(True); 

# show stream lines
#if os.path.isfile(DATA_PATH+'scalar.dat'):
#    DATA_STR=numpy.load(DATA_PATH+'scalar.dat');
#    subplot(1,1,1);
#    cstr = contour(MESHX,MESHY,DATA_STR,20);
#    title(' stream - isolines ');
#    colorbar(cstr)
#    axis('equal')
#    hold(True);

title(' potential/stream - isolines ');
#grid(True)
show()

File: run/ShowData.m
% Plotting grid

clear all
shading interp

% load grid
MESHX=load('phyGrid.meshX');
MESHY=load('phyGrid.meshY');

% show data
if exist('str_sol.dat')
    DATA_STR=load('str_sol.dat');
    iso=subplot(2,2,1:2);
        [C_str,h_str]=contour(MESHX,MESHY,DATA_STR,20);
        title(' isolines ');
        set(iso,'DataAspectRatio',[1 1 1]);
        hold on;
    str=subplot(2,2,3);
        surf(MESHX,MESHY,DATA_STR);
        title(' stream ');
        set(str,'DataAspectRatio',[1 1 1]);   
    view(2);
    save 'str_cont.dat' -ASCII -DOUBLE C_str
end
if exist('pot_sol.dat')
    DATA_POT=load('pot_sol.dat');
    iso=subplot(2,2,1:2);
        [C_pot,h_pot]=contour(MESHX,MESHY,DATA_POT,20);
        title(' isolines ');
        set(iso,'DataAspectRatio',[1 1 1]);        
        hold on;
    pot=subplot(2,2,4);
        surf(MESHX,MESHY,DATA_POT);
        title(' potential ');
        set(pot,'DataAspectRatio',[1 1 1]);
    view(2);
    save 'pot_cont.dat' -ASCII -DOUBLE C_pot
end


plottools('on');


